spyder/plugins/variableexplorer/widgets/collectionsdelegate.py

```diff
@@ -135,16 +135,17 @@ def createEditor(self, parent, option, index, object_explorer=False):
                     message.format(val_type=val_type, module=module))
                 return
             else:
-                message = _("Spyder is unable to show the variable you're"
-                            " trying to view because the module "
-                            "<tt>{module}</tt> is not ")
                 if running_in_mac_app():
-                    message += _("supported in the Spyder MacOS "
-                                 "application.<br>")
+                    message = _("Spyder is unable to show the variable you're"
+                                " trying to view because the module "
+                                "<tt>{module}</tt> is not supported in the "
+                                "Spyder MacOS application.<br>")
                 else:
-                    message += _("found in your Spyder environment. Please "
-                                 "install this package in your Spyder "
-                                 "environment.<br>")
+                    message = _("Spyder is unable to show the variable you're"
+                                " trying to view because the module "
+                                "<tt>{module}</tt> is not found in your "
+                                "Spyder environment. Please install this "
+                                "package in your Spyder environment.<br>")
```

Comment:
```suggestion
                                "package in that environment.<br>")
```

Avoid the repeated `Spyder environment`.

Reply:
I think "this environment" sounds better. Let me know if you agree to update the suggestion.